kerlescan/error.py

```diff
@@ -1,8 +1,7 @@
-from flask import current_app, jsonify
+import json
 
+from connexion.lifecycle import ConnexionRequest, ConnexionResponse
 
-def handle_http_error(error):
-    json_response = jsonify({"message": error.message})
-    response = current_app.make_response(json_response)
-    response.status_code = error.status_code
-    return response
+
+def handle_http_error(request: ConnexionRequest, exc: Exception) -> ConnexionResponse:
+    return ConnexionResponse(status_code=exc.status_code, body=json.dumps({"message": exc.message}))
```

Comment:
Isn't there a class that would return a JSON type of response? This one adds the JSON dump without a proper mime type.

Reply:
You can give MimeType to the ConnexionResponse constructor:
```
class ConnexionResponse:
    """Connexion interface for a response."""

    def __init__(
        self,
        status_code=200,
        mimetype=None,
        body=None,
        ....
    ):